.dotnet/tests/Audio/TranscriptionTests.cs

```diff
@@ -179,7 +179,7 @@ public async Task TranscriptionFormatsWork(string responseFormat)
 
                 Assert.That(segment.Id, Is.EqualTo(i));
                 Assert.That(segment.EndTime, Is.GreaterThanOrEqualTo(segment.StartTime));
-                Assert.That(segment.TokenIds, Is.Not.Null.And.Not.Empty);
+                Assert.That(segment.TokenIds.ToArray(), Is.Not.Null.And.Not.Empty);
```

Comment:
This one is fixed in main.  😊 Had to fix it as part of Friday's release and then port it back to my repo.